auth-web/src/views/auth/OrgNameAutoComplete.vue

```diff
@@ -83,8 +83,11 @@ export default class OrgNameAutoComplete extends Vue {
       try {
         const response: AutoCompleteResponse = await this.getOrgNameAutoComplete(searchValue)
         if (searchValue === this.searchValue && response?.results) {
-        // will slice results - similar to PPR
-          this.autoCompleteResults = response?.results.slice(0, ORG_AUTO_COMPLETE_MAX_RESULTS_COUNT)
+        // will slice results and show distinct results by value - similar to PPR
```

Comment:
is it tough to write a unit test case for this?
like assert duplicates are removed